### DISCUSSION TITLE
Use filters to redirect via a specific interface

### DISCUSSION THREAD

**Discussion by aik** [tinyproxy/tinyproxy#523] [Q&A]
There are tun0 (openconnect) and eth0 (main network). I need to route *.microsoft.com via tun0's IP (so it is not as simple as `ip r add` as there are many networks) and everything else via eth0. Is there a way to teach tinyproxy this?

**Comment by rofl0r** [comment]
if you control the network behind tun0 you could run a separate tinyproxy instance there, and on your localhost's tinyproxy add an upstream for microsoft.com that points to that tinyproxy's ip address. now when you connect to your localhost tinyproxy, it'll connect to the other proxy in the VPN for those requests, the rest goes out on the default interface.
